client/me/security-u2f-key/delete-item-button.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
```

Comment:
It's nice to import a specific icon for use like this:

https://github.com/Automattic/wp-calypso/blob/c66660e2ceb5f10fbbde84fd9ff3beb3b5bd1644/client/components/tinymce/plugins/insert-menu/menu-items.jsx#L9-L13

Reply:
https://github.com/Automattic/wp-calypso/pull/27626#discussion_r274470691